server/config/config.json

```diff
@@ -1,13 +1,16 @@
 {
   "development": {
-    "username": "PG_USERNAME",
-    "password": "PG_PASSWORD",
-    "database": "PG_DATABASE",
```

Comment:
Yes, the database URL would have all the info required to connect. Setting the following env variables instead of a DB URL would also work 

DB_USER=***
DB_NAME=***
DB_PASS=***